fix: clarify how to create multiple box shadows (Issue: #35534)

shashank-bairy opened the pull request:
<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #35534 
Made the required changes asked by @erictleung to resolve issue #35534 . I have tested the changes locally and its working.

shashank-bairy committed:
fix: clarify how to create multiple box shadows

ojeytonwilliams commented on curriculum/challenges/english/01-responsive-web-design/applied-visual-design/add-a-box-shadow-to-a-card-like-element.english.md:
```diff
@@ -8,7 +8,16 @@ videoUrl: 'https://scrimba.com/c/cvVZdUd'
+  <li><code>offset-y</code> (how far to push the shadow vertically from the element),</li>
+  <li><code>blur-radius</code>,</li>
+  <li><code>spread-radius</code> and</li>
+  <li>a color value, in that order.</li>
```
```suggestion
  <li><code>color</code>, in that order.</li>
```
I think this is more consistent with the rest of the bullet points.

ojeytonwilliams reviewed (commented):
Other than that last line, this looks good to me.  Thanks for the PR!

ojeytonwilliams committed:
Update curriculum/challenges/english/01-responsive-web-design/applied-visual-design/add-a-box-shadow-to-a-card-like-element.english.md

<code> tags added to 'color' property in order to make it look more consistent with the other propertes of 'box-shadow'.

Co-Authored-By: BA1RY <shashank.bairy04@gmail.com>

BA1RY commented:
Thanks for your suggestion @ojeytonwilliams . I have commited the changes you suggested.

ojeytonwilliams commented:
@BA1RY Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

RandellDawson commented:
@ojeytonwilliams I sent you a DM on Gitter.
